src/formatDate.js

```diff
@@ -3,7 +3,7 @@
 /**
  *   Time flies, standards change. Let's get rid of the routine of changing the
  * date format. Create a `formatDate` function that accepts the `date` string,
- * the old `fromFormat` array and the new `toFormat` array. Function returns
+ * the old `fromForm` array and the new `toForm` array. Function returns
```

Comment:
There is a typing error in the function description. It should be 'fromFormat' instead of 'fromForm'